app/views/bin/index.html.slim

```diff
@@ -14,63 +14,76 @@
 - if @chord_sheets.present?
   h2.title.is-3 Deleted Chord Sheets
 
-  .is-flex.is-flex-wrap-wrap.is-justify-content-center style="gap: 2rem;"
-    - @chord_sheets.each do |chord_sheet|
-      div style="border: 1px solid lightgrey; border-radius: 1rem; padding: 1rem; min-width: 40vw"
-        .is-flex.is-justify-content-space-between.is-flex-wrap-wrap style="gap: .5rem"
-          p
-            b Name:&nbsp;
-            span = chord_sheet.name
+  .mb-6 data-controller="search show-more" style="gap: 2rem;"
+    .is-flex.is-justify-content-space-between.is-align-items-center
```

Comment:
> tweak styles to make it look better (search bar and first chord are now "glued" together)

For this point I would add a `mb-4` or similar to this element. Docs: https://bulma.io/documentation/helpers/spacing-helpers/